tests/Integration/Behaviour/Features/Context/Domain/OrderFeatureContext.php

```diff
@@ -1726,4 +1726,40 @@ private function getProductTaxCalculator(int $orderId, int $productId)
 
         return $taxManager->getTaxCalculator();
     }
+
+    /**
+     * @Then the product :productReference in order :orderReference has the following prices for viewing in BO:
```

Comment:
```suggestion
     * @Then the product :productReference in order :orderReference has following prices:
```

Reply:
I want to keep "for viewing in BO", because it's a calculation that's made for display only (the real price is actually right, it doesn't change what the customer pays in the end)